tailwind.config.js

```diff
@@ -11,6 +11,9 @@ module.exports = {
         unselected: "#798393",
         button: "#F5F7FA",
         placeholder: "#E0E4E9",
+        lightBlue: "#E9F4FF",
+        darkBlue: "#004284",
+        hoverState: "#F7F9FB",
```

Comment:
May want to coord with @shannonmhong in #45 to make sure this color doesn't get added twice. I suggested the name "hover"